src/rust/e2e-tests/tests/sysmon_log_e2e_test.rs

```diff
@@ -0,0 +1,177 @@
+#![cfg(feature = "integration_tests")]
+mod test_utils;
+
+use bytes::Bytes;
+use kafka::{
+    config::ConsumerConfig,
+    test_utils::topic_scanner::KafkaTopicScanner,
+};
+use plugin_work_queue::test_utils::scan_for_plugin_message_in_pwq;
+use rust_proto::graplinc::grapl::{
+    api::{
+        event_source::v1beta1::CreateEventSourceRequest,
+        graph::v1beta1::GraphDescription,
+        pipeline_ingress::v1beta1::PublishRawLogRequest,
+        plugin_registry::v1beta1::{
+            DeployPluginRequest,
+            PluginMetadata,
+            PluginType,
+        },
+    },
+    pipeline::v1beta1::RawLog,
+};
+use test_context::test_context;
+use test_utils::context::E2eTestContext;
+use uuid::Uuid;
+
+use crate::test_utils::predicates::{
+    events_36lines_merged_graph_predicate,
+    events_36lines_node_identity_predicate,
+};
+
+mod test_fixtures {
+    use super::*;
+
+    /// Send 1 line (well, event) at a time
+    pub fn get_36_eventlog_xml_separate_lines() -> Result<Vec<String>, std::io::Error> {
+        let filename = "/test-fixtures/36_eventlog.xml"; // This path is created in rust/Dockerfile
+        let content = std::fs::read_to_string(filename)?;
+        Ok(content.lines().map(&str::to_owned).collect())
+    }
+
+    pub fn get_sysmon_generator() -> Result<Bytes, std::io::Error> {
+        std::fs::read("/test-fixtures/sysmon-generator").map(Bytes::from)
+    }
+}
+
+#[test_context(E2eTestContext)]
+#[tokio::test]
+async fn test_sysmon_log_e2e(ctx: &mut E2eTestContext) -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
My preference is to use eyre::Result for tests because just using std Result with Box<dyn std::error::Error> won't get you nearly as much, including the error's Display.

Reply:
That's great to know, I'll look into it